packages/common/src/eips/3670.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "EIP-3670",
+    "number": 3670,
+    "comment": "EOF - Code Validation",
+    "url": "https://eips.ethereum.org/EIPS/eip-3670",
+    "status": "Review",
+    "minimumHardfork": "london",
+    "requiredEIPs": [
+        3540,
+        3541
```

Comment:
Please do not add `3541` here, this should be only in `3540`, otherwise this will be redundant.

Reply:
Understood.  What about 3860?  I just noticed that EIP3670 says its requires that one as well which would notionally make this PR dependent on #1619.

Reply:
Ah, yes, then it makes sense to add here as well.

Reply:
I will finish 3860 this week, once that is merged then we should also add 3860 there (and rebase/merge `master` into #1719)